frontend/src/main/web/src/components/TableRow.js

```diff
@@ -0,0 +1,57 @@
+import React, { PropTypes } from 'react'
+import { merge } from 'lodash'
+import { Row } from './'
+
+const classes = {
+  base: {
+    bd: 'Bdb(bd1) Bdc(light)',
+    h: 'H(r2)'
+  },
+  highlight: {
+    trs: 'Trs(aeo)',
+    hover: {
+      bgc: 'Bgc(light):h'
+    }
+  },
+  selected: {
+    bgc: 'Bgc(light)',
+    custom: 'row--selected'
+  }
+}
+
+const TableRow = ({
```

Comment:
It looks like this basically just makes a `<Row/>` component. Does this mainly add extra styling to it? Might be worth mentioning that in the doc too.